<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CSCO-W-PF32ZV1R" failures="8" tests="18" name="getCustomerData" time="15.283" errors="0" timestamp="2021-10-04T18:53:42 IST">
  <testcase classname="com.java.qa.sccm.test.LoginTest" name="loginTestValidation" time="2.957"/>
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.666"/>
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.615">
    <failure type="java.lang.AssertionError" message="expected [customer_name cannot be empty] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [customer_name cannot be empty] but found [null]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.619">
    <failure type="java.lang.AssertionError" message="expected [Kindly provide valid values of customer_name] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Kindly provide valid values of customer_name] but found [null]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:94)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.602">
    <failure type="java.lang.AssertionError" message="expected [customer_id cannot be empty] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [customer_id cannot be empty] but found [null]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:99)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.596">
    <failure type="java.lang.AssertionError" message="expected [Kindly provide valid values of customer_id] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Kindly provide valid values of customer_id] but found [null]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.602">
    <failure type="java.lang.AssertionError" message="expected [short_name cannot be empty] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [short_name cannot be empty] but found [null]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:109)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.CreateCustomerTest" name="createCustomerTest" time="0.604">
    <failure type="java.lang.AssertionError" message="expected [Short Name should contain at most 15 characters.] but found [Short Name Should Contain At Most 15 Characters.]">
      <![CDATA[java.lang.AssertionError: expected [Short Name should contain at most 15 characters.] but found [Short Name Should Contain At Most 15 Characters.]
at com.java.qa.sccm.test.CreateCustomerTest.createCustomerTest(CreateCustomerTest.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="com.java.qa.sccm.test.GetCustomerDataTest" name="validateCustomerData" time="0.641"/>
  <testcase classname="com.java.qa.sccm.test.GetCustomerDataTest" name="validateCustomerData" time="0.621"/>
  <testcase classname="com.java.qa.sccm.test.GetCustomerDataTest" name="validateCustomerData" time="0.601">
    <failure type="java.lang.AssertionError" message="expected [5002] but found [5003]">
      <![CDATA[java.lang.AssertionError: expected [5002] but found [5003]
at com.java.qa.sccm.test.GetCustomerDataTest.validateCustomerData(GetCustomerDataTest.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateCustomerData -->
  <testcase classname="com.java.qa.sccm.test.UpdateCustomerTest" name="updateCustomerData" time="0.62"/>
  <testcase classname="com.java.qa.sccm.test.UpdateCustomerTest" name="updateCustomerData" time="0.597"/>
  <testcase classname="com.java.qa.sccm.test.UpdateCustomerTest" name="updateCustomerData" time="0.585"/>
  <testcase classname="com.java.qa.sccm.test.DeleteCustomerTest" name="ValidateDeleteCustomer" time="1.213"/>
  <testcase classname="com.java.qa.sccm.test.DeleteCustomerTest" name="ValidateDeleteCustomer" time="1.204"/>
  <testcase classname="com.java.qa.sccm.test.DeleteCustomerTest" name="ValidateDeleteCustomer" time="1.186"/>
  <testcase classname="com.java.qa.sccm.test.serviceDeploy.Step1Test" name="ValidationStep1" time="0.601">
    <failure type="java.lang.AssertionError" message="expected [6001] but found [6001]">
      <![CDATA[java.lang.AssertionError: expected [6001] but found [6001]
at com.java.qa.sccm.test.serviceDeploy.Step1Test.ValidationStep1(Step1Test.java:79)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- ValidationStep1 -->
</testsuite> <!-- getCustomerData -->
